test/StreamJsonRpc.Tests/MarshalableProxyTests.cs

```diff
@@ -117,10 +117,34 @@ public interface IMarshalableWithEvents : IDisposable
         event EventHandler? Foo;
     }
 
+    [RpcMarshalable]
+    [RpcMarshalableKnownSubtype(typeof(IMarshalableSubType1), subTypeCode: 1)]
+    [RpcMarshalableKnownSubtype(typeof(IMarshalableSubType2), subTypeCode: 2)]
+    public interface IMarshalableWithSubTypes : IDisposable
+    {
+        Task<int> GetAsync(int value);
+    }
+
+    public interface IMarshalableSubType1 : IMarshalableWithSubTypes
```

Comment:
Should I be more concerned about explicit interface implementations?

Reply:
There is no such thing as explicit interface implementations at the interface level. That's something determined by the implementing class. And yes, we should support those. I suspect we already do and have tests to show it. That seems like it should be orthogonal to this feature, so I don't think you're on the hook to find or add tests for it.